tools/run/bin/debug-check

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Usage:
+# # run all test cases
+# debug-check
+#
+# # run one testcase at a time
+# debug-check one_by_one
+#
+# # run with ct options
+# debug-check --case=Case0 --repeat 100
+#
+#
+lib_dir=$(dirname "$(realpath "$0")")
+
+# for compiler flag
+#export QUICER_USE_SANITIZERS=1
+
+# for using asan emulator
+export ESCRIPT_EMULATOR="${lib_dir}/erl-debug"
+
+# Since cerl returns different code:root_dir(), we need to override it here
+# Erlang_OTP_ROOT_DIR will be picked up by CMakeLists.txt
+export Erlang_OTP_ROOT_DIR=$(dirname $(dirname $(which erl)))
+
+# Set ERL_TOP for Suppressions list
+export ERL_TOP=$(cerl -noshell -eval "io:format(\"~s\", [code:root_dir()])" -s erlang halt)
+
+export QUICER_USE_SNK=1
+
+REBAR3=$(command -v rebar3)
+
+if [ $# -eq 1 ]; then
+    case $1 in
+        all)
+            escript "$REBAR3" ct
+            ;;
+        one_by_one)
+            AllTCs=$(erl -pa _build/test/lib/quicer/test/  -noshell \
+                -eval 'io:format("~p", [lists:flatten( string:join(lists:map(fun erlang:atom_to_list/1, quicer_SUITE:all()), " ") )]), halt()')
+            for tc in ${AllTCs};
+            do
+                echo "running tc $tc";
+                escript "$REBAR3" do ct --suite=test/quicer_SUITE --case="$tc";
+            done
+            ;;
+        proper)
+            escript "$REBAR3" as test proper
+            ;;
+    esac
```

Comment:
Micro nit (otherwise might fail silently in CI due to typos):

```suggestion
            ;;
        *)
            echo "Invalid option: $1"
            exit 1
            ;;
    esac
```

Reply:
(same for valgrind-check)